app/controllers/samples_status_controller.rb

```diff
@@ -27,7 +27,7 @@ def set_activity_type_selected
   end
 
   def set_activity_types
-    @activity_types = ActivityType.all.visible.sort { |a,b| a.name <=> b.name }.uniq
+    @activity_types = ActivityType.visible.alphabetical
```

Comment:
This was mostly Rubocop complaining about the sort, (suggesting sort_by(&:name) instead) but I decided a scope would be even cleaner.